roxie/udplib/udplib.hpp

```diff
@@ -82,7 +83,7 @@ class UDPLIB_API ServerIdentifier
 
 extern UDPLIB_API ServerIdentifier myNode;
 
-interface IMessagePacker : extends IInterface
+interface IMessagePacker : extends IEngineRowAllocator
```

Comment:
I'm not sure inheritance is the correct structure.  Probably cleaner to have a queryAllocator() virtual method.  May avoid the need for the include and the include path.